crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -399,6 +404,7 @@ def save_node
         :description   => :description,
         :availability_zone => :availability_zone,
         :intended_role => :intended_role,
+        :disks_roles   => :disks_roles,
```

Comment:
Style/ExtraSpacing: Unnecessary spacing detected.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.